pkg/tuple/structs.go

```diff
@@ -0,0 +1,200 @@
+package tuple
+
+import (
+	"errors"
+	"fmt"
+
+	core "github.com/authzed/spicedb/pkg/proto/core/v1"
+	"github.com/authzed/spicedb/pkg/spiceerrors"
+)
+
+const (
+	// Ellipsis is the Ellipsis relation in v0 style subjects.
+	Ellipsis = "..."
+
+	// PublicWildcard is the wildcard value for subject object IDs that indicates public access
+	// for the subject type.
+	PublicWildcard = "*"
+)
+
+// ObjectAndRelation represents an object and its relation.
+type ObjectAndRelation struct {
```

Comment:
makes me wonder if we are running `fieldalignment` linter

Reply:
I doubt it?

Reply:
Ran it locally and will address